GoogleForCreators/web-stories-wp
# Allow scaling media using scroll-wheel

## Summary

Adds scroll-wheel manipulation to media editor.

## Relevant Technical Choices

* This required adding a tiny bit of plumbing to make karma send scroll-wheel events through to puppeteer.

## To-do

* [x] Add scroll wheel listener
* [x] Add karma test

~Note that both the implementation and the test will be updated once this or #5323 is merged, whichever comes first.~ This has now been done in fe333c05.

## User-facing changes

You can now scale an image using the scroll-wheel in the media editor.

## Testing Instructions

1. Add an image
1. Double-click image to enter edit-mode
1. Use mouse scroll-wheel to change the image scale

Note that post-merge of #5323, multiple instances of scaling using mouse-wheel only happens in a single history step and can, after exiting edit-mode, be undone by pressing "undo" just once.

---



Fixes #1427
